cms/static/js/views/pages/course_outline.js

```diff
@@ -18,6 +18,8 @@ define([
                 'click .button-toggle-expand-collapse': 'toggleExpandCollapse'
             },
 
+            findElementCounter: 5000,
```

Comment:
Why `5000`? Could you add a small comment to explain the seemingly random constant value?

Reply:
5,000 refers to 5,000 milliseconds. Yes, I will add a comment to clarify 5,000 and 100.